hack/dev/kind/install.sh

```diff
@@ -24,8 +24,25 @@ export KUBECONFIG=${HOME}/.kube/config.${KIND_CLUSTER_NAME}
 export TARGET=kubernetes
 export DOMAIN_NAME=paac-127-0-0-1.nip.io
 
+if [ -z "${TEST_GITEA_SMEEURL}" ]; then
```

Comment:
I think that make setting up gosmee a requirement, i don't think it should be because in certain case some people may use this and have other way to setup forwarding.

what about just a warning?

You don't have to do this as well, but maybe [setting up a gosmee service like the way we do for gitea](https://github.com/openshift-pipelines/pipelines-as-code/blob/main/hack/dev/kind/gitea/gitea-deployment.yaml#L99-L126) but going to the controller may be a bit more easier than telling people about systemd/launchctl, what do you think?

Reply:
The warning would be better IMO 🤔 , I've faced this requirement due to:

[gitea/deploy.py](https://github.com/openshift-pipelines/pipelines-as-code/blob/main/hack/dev/kind/gitea/deploy.py#L30C4-L33)

Reply:
Ah yes right I completely get confused with something else , nevermind all good then